src/Lecture1.hs

```diff
@@ -104,7 +113,17 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+strSum :: String -> Int
+strSum str = go 0 (toNums (words str))
+       where
+        toNums :: [String] -> [Int]
+        toNums s = (map (\ x -> read x ::Int) s)
+        go :: Int -> [Int] -> Int
+        go s nums =
+           if null nums
+           then s
+           else go (s + (head nums)) (tail nums)
```

Comment:
There's already a standard function `sum` to find the sum of all elements in the list. So you don't need to write your own function 🙂